daemon/algod/api/server/v2/generated/participating/private/private_routes.yml

```
@@ -0,0 +1,16 @@
+package: private
+generate:
+  echo-server: true
+  embedded-spec: true
```

This can be turned off in the config files.
```suggestion
  embedded-spec: false
```

---

Yes, we can remove it--I've left it in since it matches what we currently have.